src/main/java/com/coveros/selenified/selenium/WaitFor.java

```diff
@@ -804,7 +804,7 @@ public int elementNotEnabled(Element element, double seconds) {
             // wait for up to XX seconds
             double end = System.currentTimeMillis() + (seconds * 1000);
             while (System.currentTimeMillis() < end) {
-                if (!is.elementPresent(element, false) || !webElement.isEnabled()) {
```

Comment:
this can be accomplished. think of this usecase:
Element is enabled on the page, element is disabled (second usage that you left)
Element is enabled on the page, element is removed (first usage that you removed)